wine-tkg-git/customization.cfg

```diff
@@ -92,7 +92,7 @@ _use_pba="true"
 _use_vkd3d="false"
 
 # dxvk - Replaces wined3d d3d10 and d3d11 with winelib DXVK for d3d10 and d3d11 translation
-_use_dxvk="false"
+_use_dxvk="true"
```

Comment:
DXVK should never be enabled by default, especially in this winelib state

Reply:
Whoops, my mistake